time/src/macros.rs

```diff
@@ -65,7 +65,7 @@ pub use time_macros::datetime;
 /// );
 /// # Ok::<_, time::Error>(())
 /// ```
-/// 
+///
```

Comment:
You'll want to run `cargo +nightly fmt` to ensure changes like this don't happen.

Reply:
Done.